README.md

```diff
@@ -2,7 +2,10 @@
 <h1 align="center">Service Mesh Patterns</h1>
 </div>
 <p align="center">
-    <img width="60%" src=".github/readme/images/logo_with_text.svg">
+<picture align="center">
+<source media="(prefers-color-scheme: dark)" srcset=".github/readme/images/service-mesh-patterns-side_white.svg" width="70%" align="center" style="margin-bottom:20px;">
+<source media="(prefers-color-scheme: light)" srcset=".github/readme/images/logo_with_text.svg" width="70%" align="center" style="margin-bottom:20px;">
```

Comment:
Nit - You might be more consistent with file naming

Reply:
sure I will keep that in mind

Reply:
This is what I would suggest:
```suggestion
<source media="(prefers-color-scheme: dark)" srcset=".github/readme/images/logo_with_text_white.svg" width="70%" align="center" style="margin-bottom:20px;">
<source media="(prefers-color-scheme: light)" srcset=".github/readme/images/logo_with_text.svg" width="70%" align="center" style="margin-bottom:20px;">
```

Optionally rename `logo_with_text.svg` to `logo_with_text_black.svg` as well if it's not breaking anything.

Reply:
Sure I will do that.

Reply:
@tomkerkhove I have made the necessary changes, please have a look